src/components/Footer/index.tsx

```diff
@@ -18,7 +18,7 @@ const Footer = (): JSX.Element => (
           target="_blank"
           href="https://trademark-policy.openjsf.org/"
           rel="noopener noreferrer"
-          aria-label="Node.js Slack Link"
+          aria-label="Node.js Trademark Policy - PDF (Opens in new tab)"
```

Comment:
Think the aria-label should be dropped here. The "Open in new tab", should be picked up by a screen reader because it is a `_blank` link, if they have that set to be announced.
The PDF part is just a current oddity of the redirect that he OpenJS site setup. It can be useful to announce that, but I'd suggest moving that to the link text, so It can also be translated in other locales

Reply:
Open in new tab is unfortunately not communicated to screen readers by the use of _blank and needs to be communicated through labels so that users don’t get disoriented